src/aws/osml/load/load_test.py

```diff
@@ -272,7 +276,7 @@ def display_image_results(job_status_dict: Dict, total_tiles_processed: int, tot
             Total Images Succeeded: {total_image_succeeded}
             Total Images Failed: {total_image_failed}
             Total Size Processed: {total_size}
-            Total Pixels Processed: {total_pixels_processed[0]}x{total_pixels_processed[1]}
+            Total Pixels Processed: {total_pixels_processed[0]*total_pixels_processed[1]}
```

Comment:
Is this actually multiplying or does it come out as a `# * #`?

Reply:
It actually multiplies - I wanted to be able conveniently extract the total pixels processed so it can feed into the peta pixel cost calc. I am probably going to add an output here which gives ``Total PetaPixels Processed:`` as well